developer-resources/Developer-Guidelines.md

```diff
@@ -162,6 +162,10 @@ code is committed into git.
 
 * For java unit tests use /src/test/java folder of the module (standard maven location), and postfix java file names with "Test".
 
+## Project Lombok
+
+Since recently in Egeria we started using [**Project Lombok**](https://projectlombok.org/) and already adopted the framework for some of the code modules.
```

Comment:
I would probably move this into the java section, perhaps as a sub-bullet.

Rather than 'since recently' How about 'Since Release 2.10 Egeria is making use of Project Lombok

I'm trying also to consider how actionable it is, and in fact whether we just drop this entirely.
We are dependent on many things as part of the build - in the pom & gradle definitions, and there's nothing for a CLI user to do. If they're coding they'll be needing to look at the pom anyway.

So maybe that short reference.. or nothing here?

Reply:
(my previous comment was not recorded somehow so posting again)

The idea was to mention it to have pointer for new developers that they can write cleanner code by using framework such as Loombok thus placed within the Coding style section. 

I will remove this for now until we finalize the discussion and decision on the standard, then document properly so new developers can be aware and make use of it.